tajo-docs/src/main/sphinx/index/how_to_use.rst

```diff
@@ -48,17 +49,17 @@ If the index is created successfully, you can see the information about that ind
 
 For more information about index creation, please refer to the above links.
 
--------------------------------------
+-----------------------------
 2. Enable/disable index scans
--------------------------------------
+-----------------------------
 
-When an index is successfully created, you must enable the index scan feature as follows:
+When an index is successfully created, you must enable index scan as follows:
```

Comment:
I suggest:
``Even though an index is successfully created, you need a further step, enabling 'index scan' as following. It is necessary because 'index scan' is disabled by default.``